shared/util/feature-flags.native.js

```diff
@@ -7,7 +7,7 @@ const ff: FeatureFlags = {
   plansEnabled: false,
   recentFilesEnabled: false,
   tabPeopleEnabled: false,
-  teamChatEnabled: false,
+  teamChatEnabled: true,
```

Comment:
forgot to revert this, right?

Reply:
i think we can leave this as the next mobile release will have teamChat